docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,102 +26,196 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
 ### Cuda
 
 The Cuda is installed by default while the cudnn is not if we use the first method. We have to write the cuda path and lib to `$PATH` and `$LD_LIBRARY_PATH`:
-```
+```shell
 export PATH=$PATH:/usr/local/cuda/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64
 ```
 Then we can use `nvcc -V` the get the version of cuda we use.
 
-### Anaconda
+### Conda
 
-We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson.
+We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson. The commands below are the example for installation, you can choose another version of Archiconda by accessing [Archiconda releases page](https://github.com/Archiconda/build-tools/releases).
+```shell
+wget https://github.com/Archiconda/build-tools/releases/download/0.2.3/Archiconda3-0.2.3-Linux-aarch64.sh
+bash Archiconda3-0.2.3-Linux-aarch64.sh
+source ~/.bashrc
+conda --version
+```
 
-After we installed the Archiconda successfully and created the virtual env correctly. If the pip in the env does not work properly or throw `Illegal instruction (core dumped)`, we may consider re-install the pip manually, reinstalling the whole JetPack SDK is the last method we can try.
+After we installed the Archiconda successfully, we need to check the tensorrt python package pre-installed by Jetpack by command below.
+```shell
+sudo find / -name tensorrt
+```
+Then you can see something like those in the terminal, take Jetson Nano as example:
+```shell
+...
+/usr/lib/python3.6/dist-packages/tensorrt
+...
+```
+The `python3.6` is the version we need to use in the conda env later.
+
+We can create the virtual env `mmdeploy` using the command below. The version of python we got from the previous step.
+```shell
+conda create -n mmdeploy python=x.x
+conda activate mmdeploy
+```
 
 ### Move tensorrt to conda env
-After we installed the Archiconda, we can use it to create a virtual env like `mmdeploy`. Then we have to move the pre-installed tensorrt package in Jetpack to the virtual env.
+Then we have to move the pre-installed tensorrt package in Jetpack to the virtual env.
 
 First we use `find` to get where the tensorrt is
-```
+```shell
 sudo find / -name tensorrt
 ```
 Then copy the tensorrt to our destination like:
+```shell
+cp -r /usr/lib/python3.6/dist-packages/tensorrt* /your/path/to/archiconda3/envs/mmdeploy/lib/python3.6/site-packages/
```

Comment:
Can we make a symbolic link instead of physically copying it?

Reply:
I use symbolic link to link it to my env, and I test import and it work. 
![image](https://user-images.githubusercontent.com/25873202/165321256-24d049ba-c149-4998-a1a2-2e158688771b.png)

![image](https://user-images.githubusercontent.com/25873202/165321327-0ab61b15-579d-44ca-b606-2feaa9ea5e6c.png)

But when I test to uninstall `tensorrt` in my env, It will delete the sorce file which is not I expect.

![image](https://user-images.githubusercontent.com/25873202/165320240-bead1be5-24c9-423e-8969-d62b5e70c1a7.png)

So I think we need to physically copying it. 😁